src/Robo/Common/EnvironmentDetector.php

```diff
@@ -349,10 +349,12 @@ public static function getAcsfDbName() {
    */
   private static function getSubclassResults($functionName) {
     if (!defined('DRUPAL_ROOT')) {
-      // Do nothing if called outside of a Drupal bootstrap.
-      return [];
+      $path = __DIR__ . "../../../../../../../docroot";
```

Comment:
Looks like you need a leading slash here. Do you suppose this path is generally consistent, or do we need to find the root in a more general way, such as my loading the BLT container and grabbing repo_root from BLT config?

Reply:
I think if there is no DRUPAL_ROOT, it's safe to assume we aren't in a Drupal bootstrap and the additional overhead of bootstrapping BLT instead makes sense. This is an incredibly hacked out approach (and I do not support it even though I wrote it). 

I think we should use the BLT container to get the repo root if DRUPAL_ROOT isn't defined. But that's just my 2 cents.

Reply:
the question @danepowell is, can we do that in a way that it doesn't happen outside the context of 'no DRUPAL_ROOT' ?

Reply:
@mikemadison13 try #4017 , I haven't tested it yet but it should solve this in a more generalizable way.